joomla/joomla-cms
# [4.0] Tinymce adopts child templates

Pull Request for Issue # .  Part of the Child template effort

### Summary of Changes
TinyMCE respects the paths for child templates

### Testing Instructions
You will need to install the front end template https://github.com/joomla/joomla-cms/files/4977972/siteparent_v2.zip from https://github.com/joomla/joomla-cms/pull/30192
The template needs to be default for all the menus
Then copy the file /templates/system/css/editor.css to /media/site/siteparent/css/editor.css

Edit an article (either backend or front end) and observe that the editor.css is loaded in the iframe of tinyMCE


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

No, feature parity, backward compatibility not broken, Logic is not radically changed, just adopted for the child able templates, also there shouldn't be any performance degradation (it's almost the same amount of commands executed prior or after this PR)
